targets/UnrealMarketplacePlugin/examplesource/ExampleProject/Plugins/PlayFab/Source/PlayFab/Classes/PfTestActor.h.ejs

```diff
@@ -116,7 +116,7 @@ public:
         FString _outputSummary;
     // A bunch of constants: load these from testTitleData.json
     UPROPERTY()
-        FString TEST_TITLE_DATA_LOC = "testTitleData.json";
+        FString TEST_TITLE_DATA_LOC = FPaths::ProjectContentDir() + TEXT("JSON/testTitleData.json");
```

Comment:
Before we do this, we need to make sure that the .gitIgnore for all Unreal repos contains exclusions for testTitleData files, so they never get submitted.

Add this to .gitIgnore, in every UE repo:

```
**/*TestTitleData.json*
**/*testTitleData.json*
```

We need to land and approve those first, before we can submit this change.

Reply:
Those two lines in .gitignore can possibly be done in one:
**/*[Tt]estTitleData.json*

---
In reply to: [256133097](https://github.com/PlayFab/SDKGenerator/pull/401#discussion_r256133097) [](ancestors = 256133097)